### PR TITLE
ok

### PR DESCRIPTION
**Pull request by TienSyForch**
merge member2

### CODE REVIEW

File: 新しいフォルダー/member2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+member2 agin merge
```
Comment (TienSyForch) [inline 新しいフォルダー/member2.txt:1]: "member2 agin merge" ok

File: member2/member2.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+member2
```
Comment (TienSyForch) [inline member2/member2.txt:1]: delete

Comment (VanTienSy) [reply member2/member2.txt:1]: ＯＫ

File: 新しいフォルダー/member2.txt (hunk 1 above)
Comment (VanTienSy) [reply 新しいフォルダー/member2.txt:1]: ＯＫ

**Review by VanTienSy — APPROVED** [review]

**Pull request merged**